mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -1895,13 +1896,33 @@ def Tosa_RescaleOp: Tosa_Op<"rescale", [Pure,
     DenseI8ArrayAttr:$shift,
     BoolAttr:$scale32,
     BoolAttr:$double_round,
-    BoolAttr:$per_channel
+    BoolAttr:$per_channel,
+    DefaultValuedOptionalAttr<BoolAttr, "false">:$input_unsigned,
+    DefaultValuedOptionalAttr<BoolAttr, "false">:$output_unsigned
   );
 
   let results = (outs
     Tosa_Tensor:$output
   );
 
+  // Custom builder that does not require optional input_unsigned and
+  // output_unsigned.
+  let builders = [
```

Comment:
To not break any existing code because mlir-tblgen will currently force the first default valued attribute to be specified (i.e. it has no default value in the constructor that takes an attribute). This is fixed by https://github.com/llvm/llvm-project/pull/118075